app/views/HomePageView.scala.html

```diff
@@ -14,25 +14,27 @@
  * limitations under the License.
  *@
 
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.html.components._
+
 @this(
-        layout: templates.Layout,
-        govukNotificationBanner: GovukNotificationBanner
+layout: templates.Layout,
+govukPhaseBanner: GovukPhaseBanner,
+notificationBanner: notificationBanner,
+paragraphBody: paragraphBody
 )
 
-@(downloadReady: Boolean = false)(implicit request: Request[_], messages: Messages)
```

Comment:
Would we be able to add the default value back onto this? Unsure of the side effects of it being removed